provers/groth16/src/verifier.rs

```diff
@@ -0,0 +1,19 @@
+use crate::{common::*, Proof, VerifyingKey};
+use lambdaworks_math::{elliptic_curve::traits::IsPairing, msm::pippenger::msm};
+
+pub fn verify(vk: &VerifyingKey, proof: &Proof, pub_inputs: &[FrElement]) -> bool {
+    // [γ^{-1} * (β*l(τ) + α*r(τ) + o(τ))]_1
+    let k_tau_assigned_verifier_g1 = msm(
```

Comment:
Here we will need something to deserialize the proof

Reply:
done in a retarded way